### DISCUSSION TITLE
What is store? How it can impact performance of app?

### DISCUSSION THREAD

**Discussion by Chintan-19** [Pavankumar-Mhaske/Ecommerce-Business-App#4] [Q&A] [answered]
Hi pavan, i saw there is the store in the application at the client and admin side of code... can you explain me what is the purpose of this store? you mentioned it impacting perfomance how ?

**Comment by Pavankumar-Mhaske** [comment, marked as answer]
👋 Hi Chintan, thank you for your interest in this repository!  💖
sure❗ The store implemented using React-Redux at both the client and admin sides ✅ serves as a centralized data management solution. Its purpose is to store and manage application-wide states efficiently. By using the React-Redux toolkit, we ensure that all states are organized in one place, making data access faster and more streamlined. This approach significantly impacts performance by reducing ❌ the need for multiple data fetches and updates across different components. It also enhances the application's responsiveness and overall user experience.

**Reply by Chintan-19** [reply]
ohh😲 that's great 🫡
